src/Sulu/Component/HttpKernel/SuluKernel.php

```diff
@@ -131,6 +132,17 @@ protected function import(RouteCollectionBuilder $routes, $glob)
         }
     }
 
+    private function glob($confDir, $pattern)
+    {
+        if (\defined('GLOB_BRACE')) {
```

Comment:
We should always use the GlobResource class of symfony. It will use glob function if available and will fallback to a custom implementation itself.

/cc @danrot

Reply:
Removed it